content/docs/developing-operators/tasks.md

```diff
@@ -47,6 +47,27 @@ tasks:
 
 **Note:** deleting non-existing resources is always successful. As of version 0.9.0, KUDO will not wait for the resource to be actually removed and will finish the task when the API server accepts the deletion request. So in case of Pods, Kubernetes imposes a default graceful termination period of 30 seconds, however, a delete-task will be done before that. KUDO 0.9.0 will not wait for [resource finalizers](https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/) should they exist.
 
+
+## Toggle-Task
+
+Developing featrues for operators sometimes requires enabling and disabling resources.
+A toggle-task runs an apply-task or delete-task based on a parameter boolean value.
+Let's add a service for an operator based on the parameter `enable-service`:
+
+```yaml
+tasks:
+  - name: app-service
+    kind: Toggle
+    spec:
+      parameter: enable-service
+      resources:
+        - service.yaml
+```
+
+Based on the `enable-service` value the task will either `apply` or `delete` the resources defined in `templates/service.yaml`
+
+If the `enable-service` parameter evaluates to `true` the task named `app-service` will create a service resource defined in `templates/service.yaml`. In case the `enable-service` parameter evaluates to `false`, the task named `app-service` will delete the service resource defined in `templates/service.yaml`.
+
```

Comment:
It would be useful to note that the parameter must be defined in the params.yaml file.

Reply:
...and mention what happens if the parameter type is not boolean.

Reply:
added the param requirement